sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 3.3.0-beta.1 (Unreleased)
+
+### Features Added
+-  Enabled support for logging [personally identifiable information](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/PII), required for customer support through the `enableSupportLogging` option on `loggingOptions` under `TokenCredentialOptions`.
```

Comment:
@scottaddie anything else you want added here?

Reply:
This looks good to me. Thanks for checking.